odxtools/basevariantmatcher.py

```diff
@@ -0,0 +1,35 @@
+# SPDX-License-Identifier: MIT
+from typing import List
+
+from .diaglayers.basevariant import BaseVariant
+from .variantmatcher import VariantMatcher
+
+
+class BaseVariantMatcher(VariantMatcher):
```

Comment:
What's the point of this class? it has no logic, VariantMatcher could be used directly